hardware_applications/README.md

```diff
@@ -0,0 +1,71 @@
+# Software Training Hardware Applications #
+
+These files contain source code for the hardware applications demonstrated in the RoboJackets Software Training program.
+
+These demos utilize the [Software Training Support Library (STSL)](https://github.com/RoboJackets/stsl) developped by RoboJackets.
+
+## How to Build the Demos ##
+
+In our training sessions, we'll be using [JetBrains' CLion IDE](https://www.jetbrains.com/clion/). Below you can find instructions on how to build the hardware application demos both with and without this IDE.
+
+### Using CLion ###
+
+1. Open CLion
+
+2. If you have another project open, select File->Open...
+
+   If you see the CLion welcome dialog, select Open Project
+   
+3. Once the project loads, select Run->Build
+
+At this point, you can select a specific demo to run using the drop down and other controls in the top right of the IDE window.
+
+## How to Run a Demo ##
+
+1. In a terminal, navigate to the hardware_applications directory.
+
+2. Create a _build_ directory and navigate into it.
+
+   ```mkdir build && cd build```
+   
+3. Configure the project with CMake
+
+   ```cmake ..```
+   
+4. Build all demos
+
+   ```make```
+
+## How to Rebuild the STSL ##
+
+Binaries and headers for the STSL are provided in the STSL folder. These should work for most circumstances. If you experience problems building the demos with the provided binaries, you may need to rebuild the STSL from source.
```

Comment:
Expand on the what STSL means and what it is for here. This will be their first introduction to it.

Reply:
This is covered by the STSL readme linked to above in this readme.